### ISSUE TITLE
Yolov8 OBB rust onnxruntime example missing

### ISSUE THREAD

**Issue by DawnMagnet** [ultralytics/ultralytics#14382]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

examples/YOLOv8-ONNXRuntime-Rust said we have OBB support, but there is not, when running the example, something crashed
```text
thread 'main' panicked at src/ort_backend.rs:171:34:
not yet implemented: "obb" is not supported for now!
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

and code around that line are
```rust
Some(task) => match task.as_str() {
                            "classify" => YOLOTask::Classify,
                            "detect" => YOLOTask::Detect,
                            "pose" => YOLOTask::Pose,
                            "segment" => YOLOTask::Segment,
                            x => todo!("{:?} is not supported for now!", x),
                        },
```
which don't have obb support

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
Hello,

Thank you for bringing this to our attention. It appears that the current implementation of the YOLOv8-ONNXRuntime-Rust example does not yet support Oriented Bounding Boxes (OBB), as indicated by the error message and the code snippet you provided.

To help us address this issue more effectively, could you please provide a minimal reproducible example? This will allow us to better understand the context and reproduce the issue on our end. You can find guidelines on how to create a minimal reproducible example [here](https://docs.ultralytics.com/help/minimum_reproducible_example).

Additionally, please ensure that you are using the latest versions of the packages involved, as updates may contain important fixes and improvements.

We appreciate your patience and understanding as we work to resolve this. If you have any further questions or need additional assistance, feel free to ask!

**Comment by glenn-jocher** [comment]
@DawnMagnet yes you are right, the OBB code is missing. If you'd like to help by contributing a PR here, that would be super helpful!

**Comment by jamjamjon** [comment]
@glenn-jocher @DawnMagnet OBB has long been supported, as described in the [README](https://github.com/ultralytics/ultralytics/tree/main/examples/YOLOv8-ONNXRuntime-Rust#recently-updated). Due to some file restrictions in the Ultralytics project, the OBB demo has been placed in another project: [usls](https://github.com/jamjamjon/uslso). You can find the YOLOv8-OBB demo at: [yolo-demo](https://github.com/jamjamjon/usls/tree/main/examples/yolo).

![obb-2](https://github.com/user-attachments/assets/8838140b-3709-4d12-a3c2-07ee1e5349e8)

**Comment by glenn-jocher** [comment]
@jamjamjon thank you for pointing that out. The OBB support for YOLOv8 in the ONNXRuntime-Rust example is indeed mentioned in the README. However, it appears that the implementation is currently missing. We recommend checking the latest package versions to see if the issue persists. If you continue to encounter problems, please feel free to provide more details, and we will assist you further.
